src/shared/components/plots/BoxScatterPlot.tsx

```diff
@@ -616,6 +616,23 @@ export default class BoxScatterPlot<D extends IBaseBoxScatterPlotPoint> extends
         );
     }
 
+    @autobind
+    private getTooltip() {
```

Comment:
how come not computed?  i guess observer will prevent re render?

Reply:
@alisman right. observer needs a function ()=>ReactElement